cvmfs/cvmfs_server

```diff
@@ -2026,6 +2027,38 @@ Alias /cvmfs/${name} ${storage_dir}
 EOF
 }
 
+create_apache_config_for_global_info() {
+  local conf_file="cvmfs.info.conf"
+
+  ! has_apache_config_file $conf_file || return 0
+
+  # TODO(rmeusel): make this a bit more flexible
+  local storage_dir="${DEFAULT_LOCAL_STORAGE}/info"
+
+  # TODO(rmeusel): fine-tune this configuration file
+  create_apache_config_file $conf_file << EOF
+# Created by cvmfs_server.  Don't touch.
+
+KeepAlive On
+Alias /cvmfs/info ${storage_dir}
+<Directory "${storage_dir}">
+    Options -MultiViews
+    AllowOverride Limit
+    $(get_compatible_apache_allow_from_all_config)
+
+    EnableMMAP Off
+    EnableSendFile Off
+
+    FileETag None
+
+    ExpiresActive On
+    ExpiresDefault "access plus 3 days"
+    ExpiresByType text/html "access plus 5 minutes"
+    ExpiresByType application/x-cvmfs "access plus 2 minutes"
+</Directory>
+EOF
+}
+
```

Comment:
The current apache config used for cvmfs repositories is not ideal but it works, so I figured that the easiest thing would be to just use the same one, changing only the minimum necessary.  That's why I made a create_apache_config_core() function in my compat/update_repositories_info script, assuming something like that would be used for both purposes.